src/js/components/Settings/VoterEmailAddressEntry.jsx

```diff
@@ -531,50 +531,50 @@ class VoterEmailAddressEntry extends Component {
             </FirstRowPhoneOrEmail>
             <SecondRowPhoneOrEmail>
               {isPrimaryEmailAddress ? (
-                <div key={`${voterEmailAddressFromList.email_we_vote_id}-internal`}>
+                <SecondRowPhoneOrEmailDiv>
                   <span>
                     Primary
-                    <span style={{ paddingRight: '112px' }}>&nbsp;</span>
                   </span>
-                  <OverlayTrigger
-                    placement="right"
-                    overlay={(
-                      <Tooltip id="tooltip-top">
-                        Cannot remove primary email. Please add another primary email before removing this one.
-                      </Tooltip>
-                    )}
-                  >
-                    <TrashCan>
-                      <span
-                        // className="u-gray-darker"
-                        style={{ color: '#757575' }}
-
-                      >
-                        <Delete />
-                      </span>
-                    </TrashCan>
-                  </OverlayTrigger>
-                </div>
+                  <span>
+                    <OverlayTrigger
+                      placement="right"
+                      overlay={(
+                        <Tooltip id="tooltip-top">
+                          You must add a new primary email before removing this one.
+                        </Tooltip>
+                      )}
+                    >
+                      <div>
+                        <span
+                          className="u-gray-border"
+                        >
+                          <Delete />
+                        </span>
+                      </div>
+                    </OverlayTrigger>
+                  </span>
+                </SecondRowPhoneOrEmailDiv>
               ) : (
-                <div key={`${voterEmailAddressFromList.email_we_vote_id}-internal`}>
+                <SecondRowPhoneOrEmailDiv key={`${voterEmailAddressFromList.email_we_vote_id}-internal`}>
                   <span
                     className="u-link-color u-cursor--pointer u-no-break"
                     onClick={() => this.setAsPrimaryEmailAddress(voterEmailAddressFromList.email_we_vote_id)}
                   >
                     Make Primary
-                    <span style={{ paddingRight: '65px' }}>&nbsp;</span>
                   </span>
                   {allowRemoveEmail && (
-                    <TrashCan>
-                      <span
-                        className="u-link-color u-cursor--pointer"
-                        onClick={() => this.removeVoterEmailAddress(voterEmailAddressFromList.email_we_vote_id)}
-                      >
-                        <Delete />
-                      </span>
-                    </TrashCan>
+                  <span>
```

Comment:
Check this with `npm run lint` but I think this whole block needs to be indented from the line above.